src/main/resources/db/changelog/changes/v1.1.0/sql/entity-type-definitions/insert-organization-definition.sql

```diff
@@ -0,0 +1,294 @@
+                            "visibleByDefault": false
+                 },
+                {
+                            "name": "accounting_code",
+                            "dataType": {
+                              "dataType": "stringType"
+                                   },
+                            "filterValueGetter": "${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb ->> ''erpCode''::text)",
+                            "valueFunction": "${tenant_id}_mod_organizations_storage.f_unaccent(:value)",
+                            "valueGetter": "org.jsonb->>''erpCode''",
+                            "visibleByDefault": false
+                 },
+                 {
+                            "name": "payment_method",
+                            "values": [
+                                             {
+                                                   "label": "Credit Card",
+                                                   "value": "Credit Card"
+                                              },
+                                              {
+                                                   "label": "Cash",
+                                                   "value": "Cash"
+                                              },
+                                              {
+                                                    "label": "Physical Check",
+                                                    "value": "Physical Check"
+                                              },
+                                              {
+                                                   "label": "EFT",
+                                                   "value": "EFT"
+                                              },
+                                              {
+                                                    "label": "Deposit Account",
+                                                    "value": "Deposit Account"
+                                              }
+                                       ],
+                            "dataType": {
+                              "dataType": "stringType"
+                                   },
+                            "valueGetter": "org.jsonb->>''paymentMethod''",
+                            "visibleByDefault": false
+                 },
+                {
+                            "name": "discount_percent",
+                            "dataType": {
+                              "dataType": "numberType"
+                                   },
+                            "valueGetter": "org.jsonb->>''discountPercent''",
+                            "visibleByDefault": false
+                 },
+                 {
+                            "name": "claiming_interval",
+                            "dataType": {
+                              "dataType": "integerType"
+                                   },
```

Comment:
formatting looks off here, too...may be worthwhile to copy/paste the whole blob through https://jsonbeautifier.org/ or similar